src/main/java/org/opentripplanner/routing/algorithm/RoutingWorker.java

```diff
@@ -44,19 +45,23 @@
     private final RoutingRequest request;
     private final Router router;
     private FilterTransitWhenDirectModeIsEmpty emptyDirectModeHandler;
-    private final ZonedDateTime searchStartTime;
+
+    /**
+     * Transit service time-zero. Usually midnight before the request dateTime, but NOT on days
+     * witch has DST adjustments.
+     */
+    private final ZonedDateTime searchTransitTimeZero;
```

Comment:
Something like datum or reference might be also good names, I'm not really sure if this is descriptive enough in itself

Reply:
Maybe `transitSearchTimeZero` is better. `Datum` would work, but it is not in my vocabulary. I will update the JavaDoc.